x-pack/test/search_sessions_integration/tests/apps/discover/sessions_in_space.ts

```diff
@@ -19,16 +20,27 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     'visChart',
     'security',
     'timePicker',
+    'searchSessionsManagement',
   ]);
   const browser = getService('browser');
   const searchSessions = getService('searchSessions');
+  const kibanaServer = getService('kibanaServer');
+  const toasts = getService('toasts');
 
-  // FLAKY https://github.com/elastic/kibana/issues/112913
```

Comment:
Test was skipped because of https://github.com/elastic/kibana/issues/113520
Adding a workaround for flakiness and also adjusting a test to go throng mgmt page so that it goes through URL service flow